packages/travix-ui-kit/components/slidingPanel/slidingPanel.js

```diff
@@ -132,6 +133,62 @@ export default class SlidingPanel extends Component {
     const panelClassName = classnames(getClassNamesWithMods('ui-sliding-panel', panelMods), className);
     const overlayClassName = getClassNamesWithMods('ui-sliding-panel-overlay', overlayMods);
 
+    let titleContent = title ? (
+      <SlidingPanelHeader
+        leftBlock={leftBlock}
+        onBackButtonClick={onBackButtonClick}
+        rightBlock={rightBlock}
+        useDefaultLeftBlock={useDefaultLeftBlock}
+      >
+        {title}
+      </SlidingPanelHeader>
+    ) : null;
+
+    let subheaderContent = subheader ? (
+      <div className="ui-sliding-panel__subheader">
+        {subheader}
+      </div>
+    ) : null;
+
+    let footerContent = footer ? (
+      <SlidingPanelFooter>
+        {footer}
+      </SlidingPanelFooter>
+    ) : null;
+
+    const mainContent = [];
+
+    /*
+    ** SlidingPanel migration state. This validation will handle title, subheader and footer
+    ** parameters and will support SlidingPanelHeader and SlidingPanelFooter. React components
+    ** will overides any property definition.
+    **
+    ** We also will encapsulate all components using SlidingPanelContent as a temporary solution.
+    ** After the migration process ends it will be removed and every child will have to define it if needed.
+    */
+
+    Children.forEach(children, (child) => {
```

Comment:
Why do we need this?

So, we have to fix (attach to top and bottom) header and footer components, but the content should have a normal scroll behavior.

But our component should receive almost everything as child, so we have to map all children to put Header and Footer in the right spot and the content inside our wrapper. 

Eg: 

```jsx
<SlidingPanel>
  <SlidingPanelHeader>Title</SlidingPanelHeader>
  <SlidingPanelContent>Hello Content</SlidingPanelContent>
  <SlidingPanelFooter>Footer</SlidingPanelFooter>
<SlidingPanel>
```

Will be mapped to this:
```html
<div class="sliding-panel">
  <div class="sliding-panel-header">Title</div> <!-- fixed on top -->
  <div class="sliding-panel-wrapper"> <!--  This guy knows how to handle scroll -->
    <div class="sliding-panel-content"> <!-- And this guy adds padding -->
      Hello Content
    </div>
  </div>
  <div class="sliding-panel-footer">Footer</div>
<div>
```